client/rest/resources/rest.json

```diff
@@ -40,7 +40,8 @@
     "pageSizeDefault": 20,
     "maxConnectionAttempts": 5,
     "baseRetryDelay": 500,
-    "connectionPoolSize": 10
+    "connectionPoolSize": 10,
+    "connectionTimeout": 10000
```

Comment:
I guess we could make it much smaller, say 500ms (cause during startup, rest should wait for mongo to boot anyway...)

is there any downside of setting it to much lower value/

Reply:
My only concern was with startup. On slower nodes, database startup will take longer.  But should be ok since REST will retry the connection.